src/utils/constants/wallets.ts

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
```

Comment:
It looks like indents are 4 rather than 2. It should be 2 according to the convention.